index.bs

```diff
@@ -447,6 +447,8 @@ When this method is invoked, the user agent MUST execute the following algorithm
     unless the {{ScopedCredentialOptions/attachment}} member of {{options}} is <a>present</a>. In that case, let |attachment|
     be {{ScopedCredentialOptions/attachment}}, and add the authenticator to |currentlyAvailableAuthenticators| if its attachment
     modality matches |attachment|.
+    - If no authenticator is available, reject |promise| with a <a>DOMException</a> whose name is "NotFoundError", and 
```

Comment:
Implementer question: what constitutes "authenticator not available" for this purpose? For NFC authenticators in particular, would the RP have to call the function in a loop until the user taps the authenticator on the reader? I understand this would be implementation specific for the platform, but some thought about sane implementation choices would be useful.